B_T_BC_: [-0.025809397513,
           0.025809397513,
           0.002500000000]
R_BC_:   [-0.707106781187,-0.699953260237,0.100326633967,
         -0.707106781187,0.699953260237,-0.100326633967,
         0.000000000000,-0.141883286424,   -0.989883]
cable_length_: 0.301
distance_tag_CoG: 25.7e-3
